scripts/release.sh

```diff
@@ -108,20 +110,32 @@ while [ "$#" -gt 0 ]; do
       DEBUG="yes"
       shift
       ;;
+    --coverage)
+      COVERAGE="yes"
+      shift
+      ;;
     *)
       echo "Unknown option: $1"
       exit 1
       ;;
   esac
 done
 
+if [ -n "$COVERAGE" ] && [ -n "$DEBUG" ]; then
```

Comment:
Why not ?

Reply:
see https://github.com/openebs/mayastor/pull/1007 which you approved